macosx/Snes9x/AppDelegate.m

```diff
@@ -42,25 +31,25 @@ - (void)applicationDidFinishLaunching:(NSNotification *)aNotification {
     [self setupDefaults];
     [self importRecentItems];
 
-    NSWindow *window = [[NSWindow alloc] initWithContentRect:s9xView.frame styleMask:NSWindowStyleMaskTitled|NSWindowStyleMaskClosable|NSWindowStyleMaskMiniaturizable|NSWindowStyleMaskResizable backing:NSBackingStoreBuffered defer:NO];
+    NSWindow *gameWindow = [[NSWindow alloc] initWithContentRect:s9xView.frame styleMask:NSWindowStyleMaskTitled|NSWindowStyleMaskClosable|NSWindowStyleMaskMiniaturizable|NSWindowStyleMaskResizable backing:NSBackingStoreBuffered defer:NO];
```

Comment:
Good rename.

Reply:
Intended to differentiate from the preferences window now.